### PR TITLE
HOCS-4064 Add BF QA response screen

### PR DESCRIPTION
**Pull request by gardnersam**

### CODE REVIEW

**Review by LiamMacP — CHANGES_REQUESTED** [review]
Can we use default flows here for the direction check and the valid check in BF_QA?

**Comment by gardnersam** [comment]
@LiamMacP updated with default flows

File: src/test/java/uk/gov/digital/ho/hocs/workflow/processes/BF_QA.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    @Test
+    public void testReject(){
+        when(process.waitsAtUserTask("Validate_Input"))
+                .thenReturn(task -> task.complete(withVariables("valid", true, "BfQaResult", "Reject")));
+
+        when(process.waitsAtUserTask("Validate_Reject_reason"))
+                .thenReturn(task -> task.complete(withVariables("valid", false, "DIRECTION", "BACKWARD")))
+                .thenReturn(task -> task.complete(withVariables("valid", false, "DIRECTION", "FORWARD")))
+                .thenReturn(task -> task.complete(withVariables("valid", false, "DIRECTION", "FORWARD")))
```
Comment (LiamMacP) [inline src/test/java/uk/gov/digital/ho/hocs/workflow/processes/BF_QA.java:63]: Same as L62 - is this required?

Comment (gardnersam) [reply src/test/java/uk/gov/digital/ho/hocs/workflow/processes/BF_QA.java]: No, not sure how that snuck in there, I've taken it out

**Review by LiamMacP — APPROVED** [review]
Approved - changes look good to me! Nice work 🙂

**Pull request merged**